plugins/CorePluginsAdmin/Commands/InstallPlugin.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CorePluginsAdmin\Commands;
+
+use Piwik\Config as PiwikConfig;
+use Piwik\Container\StaticContainer;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugin\Manager;
+use Piwik\Plugins\CorePluginsAdmin\PluginInstaller;
+use Piwik\Plugins\Marketplace\Marketplace;
+
+/**
+ * plugin:install console command.
+ */
+class InstallPlugin extends ConsoleCommand
+{
+    protected function configure(): void
+    {
+        $this->setName('plugin:install-or-update');
+        $this->setDescription('Install or update a plugin.');
+        $this->addOptionalArgument('plugin', 'The name of the plugin you want to install or update. Multiple plugin names can be specified separated by a space.', null, true);
+    }
+
+    protected function doExecute(): int
+    {
+        PiwikConfig::getInstance()->checkConfigIsWritable();
+
+        $input = $this->getInput();
+        $output = $this->getOutput();
+        $pluginManager = Manager::getInstance();
+
+        if (!Marketplace::isMarketplaceEnabled()) {
+            $output->writeln(sprintf("<error>Marketplace is not enabled, can't install or update plugins.</error>"));
+            return self::FAILURE;
+        }
+
+        $pluginNames = $input->getArgument('plugin');
+
+        foreach ($pluginNames as $pluginName) {
+            try {
+                $this->installPlugin($pluginName);
+                $output->writeln(sprintf("Installed plugin <info>%s</info>", $pluginName));
```

Comment:
@jsantos42 We could make a difference here between installing and updating. If the plugin was installed before, the plugin might have been updated. And if we want to be super accurate we could check if the installed version actually changed. If it doesn't actually nothing happened as there was no update available.
If you don't have time to put some extra effort into this, it's also fine to just update the text to `Installed or updated plugin...` for now.
Otherwise this PR looks good and is almost ready to merge

Reply:
```suggestion
                $output->writeln(sprintf("Installed or updated plugin <info>%s</info>", $pluginName));
```

Reply:
I've now changed the text only, so we can get that merged. Feel free to create a follow up PR if you want.

Reply:
@sgiehl sorry for getting back to you only now, I was on vacation. Thank you for fixing that on my behalf! And yes, I might address that on a future PR indeed  🙌